diff --git a/bootstrap.conf b/bootstrap.conf
index caafe0c..dbbe087 100644
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -133,6 +133,11 @@ avoided_gnulib_modules=$avoided_gnulib_modules"\
 avoided_gnulib_modules=$avoided_gnulib_modules"\
  --avoid=update-copyright-tests\
 "
+# This test fails when built with clang and sanitizers.
+avoided_gnulib_modules=$avoided_gnulib_modules"\
+ --avoid=memchr-tests\
+ --avoid=strncpy-tests\
+"
 # We use mcel-prefer.
 avoided_gnulib_modules=$avoided_gnulib_modules"\
  --avoid=mbiterf\
